py3status/modules/google_calendar.py

```diff
@@ -0,0 +1,438 @@
+# -*- coding: utf-8 -*-
```

Comment:
Remove the binary file above: `py3status/modules/.google_calendar.py.swp`. 👍

Reply:
Done.

Reply:
Remove the binary file above: `py3status/modules/.google_calendar.py.swo`. 👍